src/publish.ts

```diff
@@ -55,12 +56,18 @@ const publish = async ({ extensionId, target, asset }: PluginConfig) => {
   if (!publishRes.status.includes('OK')) {
     const errors: SemanticReleaseError[] = []
     for (let i = 0; i < publishRes.status.length; i += 1) {
-      const message = publishRes.statusList[i]
       const code = publishRes.status[i]
-      const err = new SemanticReleaseError(message, code)
-      errors.push(err)
+      const message = publishRes.statusDetail[i]
+      if (code.includes('WARNING')) {
+        logger.log('%s: %s', code, message)
```

Comment:
I think it would be more readable to use template literals instead:
```suggestion
        logger.log(`${code}: ${message}`)
```